documentation/IDEasy-contribution-getting-started.adoc

```diff
@@ -12,7 +12,7 @@ This will be addressed under xref:Contribution[Contribution].
 
 == Contribution
 
-To begin, read through the https://github.com/devonfw/.github/blob/master/CONTRIBUTING.adoc[Contribution Guidelines], which you should be sure to follow.
+To begin, read through the https://github.com/devonfw/.github/blob/master/CONTRIBUTING.asciidoc[Contribution Guidelines], which you should be sure to follow.
```

Comment:
Might have to be this URL instead: https://github.com/devonfw/IDEasy/blob/main/documentation/IDEasy-contribution-rules-and-guidelines.adoc